setup/git.md

```diff
@@ -0,0 +1,23 @@
+# Git and Github
+
+## Git
+
+Git is a version control software, which allows you to save your code in steps, like different drafts for an essay. It also allows for branches, which could represent someone else's edits to your paper, which then you merge into your main paper.
+
+For this tutorial, git and Github are not nessicary, but highly recommended. You can download git from: https://git-scm.com/.
+
+## Github
+
+Github is an online service that allows people to store their code online for free. Github uses the git program for this. You can create an account, and once that is complete, navigate to https://github.com/fabricMC/fabric-example-mod. This example mod created by fabric is template for your mods, with everything setup.
```

Comment:
I'd hyperlink the example mod, so like `[The Fabric Example Mod](https://github.com/fabricMC/fabric-example-mod)`